utils/manifests/definitions.go

```diff
@@ -0,0 +1,35 @@
+package manifests
+
+const workloadDefinitionTemplate = "{\"apiVersion\":\"core.oam.dev/v1alpha1\",\"kind\":\"WorkloadDefinition\",\"metadata\":{\"name\":\"\"},\"spec\":{\"definitionRef\":{\"name\":\"\"}}}"
```

Comment:
Please use OAM struct here.

Reply:
Addressed in https://github.com/layer5io/meshkit/pull/97/commits/123bac544df9da2fe333cfcb0fab1eda085134ed